File: thriller-4k-remaster/NightWindow.java
import java.time.ZoneId;
import java.time.ZonedDateTime;

public final class NightWindow {
    private static final ZoneId ZONE = ZoneId.of("America/Santiago");

    public static void main(String[] args) {
        ZonedDateTime now = ZonedDateTime.now(ZONE);
        System.out.printf("Thriller 4K Remaster — %s%n", now);
    }
}

File: thriller-4k-remaster/NightWindow.cpp
#include <iostream>
#include <string>

struct NightWindow {
    std::string timezone = "America/Santiago";
    std::string start = "20:00";
    std::string end = "07:15";
};

int main() {
    const NightWindow session;
    std::cout << "Thriller 4K Remaster | " << session.start << "-" << session.end
              << " " << session.timezone << '\n';
    return 0;
}
